data-imports/clean_utils.py

```diff
@@ -139,7 +139,9 @@ def clean_bbl(table, boro, block, lot):
     return sql + '''
     UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
     UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {block} = preg_replace('/[^0-9]/', '', {block});
     UPDATE {table} SET {lot} = REPLACE({lot}, ' ', 0);
+    UPDATE {table} SET {lot} = preg_replace('/[^0-9]/', '', {lot});
```

Comment:
That's a good point, it's not equivalent to your new cleaning. However, it's doing something pretty odd - replacing all spaces with 0s. I would assume this would only be correct if numbers are consistently entered with spaces instead of 0s; if spaces are accidents, then we want to actually remove them entirely. The latter is what I'd expect, but we should investigate if the former ever applies.
I'd prefer we figure this out now; however, if you don't have much time and would rather finish this PR up, we can file an issue for it and deal with it later.